modules/serverless-ossm-secret-filtering-net-istio.adoc

```diff
@@ -50,9 +46,41 @@ spec:
         sidecar.istio.io/rewriteAppHTTPProbers: "true"
       name: autoscaler
 ----
-<1> Adding this annotation injects an environment variable, `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=true`, to the `net-istio` controller pod.
-+
-[NOTE]
-====
-This annotation is ignored if you set a different value by overriding deployments.
-====
+
+You can disable the secret filtering by setting `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=false` via `workloads`in the `KnativeServing` custom resource (CR).
```

Comment:
```suggestion
You can disable the secret filtering by setting `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=false` via `workloads` in the `KnativeServing` custom resource (CR).
```

Reply:
Thank you fo capturing this typo. Updated!